ckanext/charts/assets/js/charts-render-chartjs.js

```diff
@@ -23,3 +23,77 @@ ckan.module("charts-render-chartjs", function ($, _) {
         }
     };
 });
+
+
+/*!
+  * chartjs-adapter-moment v1.0.0
```

Comment:
describe what are we using it for here

Reply:
also, how is it related to https://linkdigital.jira.com/browse/LLCAXCHZF-56 ?
I thought this task was about the `More info` button, so the PR should contain work related to only this button.